tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -415,17 +425,33 @@ public TestSetup apply(Long aLong) throws Throwable {
           final ManualSubscriberWithErrorCollection<T> sub2 = new ManualSubscriberWithErrorCollection<T>(env);
           env.subscribe(processor, sub2);
 
-          sub1.request(1);
-          expectRequest();
-          final T x = sendNextTFromUpstream();
-          expectNextElement(sub1, x);
-          sub1.request(1);
+          final Exception ex = new RuntimeException("Test exception");
```

Comment:
Since most devs probably won't ever notice this method, or know that they should enable/disable it—what do you think about including a hint about it in case the tests which use it fail?